### DISCUSSION TITLE
4x5 modification for smaller printers (Prusa mini)

### DISCUSSION THREAD

**Discussion by Egorarchitec** [edgarkech/scalable_field_camera#14] [Show and tell]
Hi I found this project and was ready to try to print it right away, but unfortunately I have Prusa Mini and it only takes 180x180x180mm models. This one is bit too big for that . So I am currently trying to modify the 4x5 models to divide the biggest parts into smaller so it can fit onto smaller print beds and look stylish at the same time (not just split and glued together). I will try to post updates if I will succeed (or not). Right now I printed test joints and printing the lid (it is the hardest to modify because it is so thin!). I really like the connection of the parts, it feels sturdy enough (full size print will show how sturdy it really is though). I might modify some parts as well (thinking about adding a screw hole for tripod crew instead of current solution)

If some of you will be interested in that, I will upload the models as well

![ViewCapture20230425_203858](https://user-images.githubusercontent.com/66465822/234374063-2a23f8b1-3133-454f-aefe-755ccf2a2972.jpg)
![ViewCapture20230425_203929](https://user-images.githubusercontent.com/66465822/234374073-a1b19be5-a09c-411c-a709-f9f97eef8093.jpg)
![20230425_204825](https://user-images.githubusercontent.com/66465822/234374668-d50884a8-fb30-4040-af61-420a7d8c84b2.jpg)
![20230425_204829](https://user-images.githubusercontent.com/66465822/234374684-09ee3f33-1db3-43bc-a62b-970f1ecfb0c8.jpg)
![20230425_204832](https://user-images.githubusercontent.com/66465822/234374690-609d3fa3-5f09-4d56-8ceb-d8f1caec501a.jpg)

**Comment by edgarkech** [comment]
Interesting approach - I am curious to see if this works without sacrificing stability or functionality. 
My initial mockup version was printed on an anycubic mega with 200x200 printing bed which worked good enough - for 4x5. But as I wanted to scale up to 5x7 I never wasted thoughts on splitting up parts but instead just taking the chance to upgrade my hardware ;-)
just pure interest about your process:
Are you modifying the SCAD files or do you generate a 3D model from the stock SCADs and modifying with another software?

**Reply by Egorarchitec** [reply]
Thanks for a reaction. I will definitely share the results.
I am not introduced into SCAD, but pretty familiar with rhino 3d. So i just imported SLT files and edit 3D model there

**Comment by Egorarchitec** [comment]
Hi! Coming with an update. After initial idea to just modify the case a bit I decided to create my own design from scratch in Fusion 360. I am adding some screenshots and photos of printed prototype. I am dealing with light leaks on the back, but i think I solved it with velvet paper around the seams.
![Architec_4x5_camera v261 (1)](https://github.com/edgarkech/scalable_field_camera/assets/66465822/e4cecbbd-2571-4aec-9834-a5ce4fc87513)
![Architec_4x5_camera v3472 (1)](https://github.com/edgarkech/scalable_field_camera/assets/66465822/4337ed34-ca9c-42a9-9f0c-219707919ee2)
![20230914_171559](https://github.com/edgarkech/scalable_field_camera/assets/66465822/da1223c6-f6e1-4f2d-83d6-1ab4d711da4a)
![20230914_171617](https://github.com/edgarkech/scalable_field_camera/assets/66465822/72928482-c7b2-426a-b2e8-4c83dfc9c679)
![20230918_182046](https://github.com/edgarkech/scalable_field_camera/assets/66465822/f546756a-21fa-4e9c-99a4-4e5bfaa6232f)
![20230927_192525](https://github.com/edgarkech/scalable_field_camera/assets/66465822/503fdedc-bc2c-4544-996b-c08c5616a042)

The photo taken on this camera
![NSND_02 (Velké)](https://github.com/edgarkech/scalable_field_camera/assets/66465822/93e0103a-afdb-4ab7-a07b-702404dc134f)

**Comment by edgarkech** [comment]
So first of all I appreciate your efforts - while such a field camera looks quite simple at first sight, there are some challenges as soon as you try to design AND print one by yourself ;-)
It's something you can be proud of, especially if it works as expected and checks all the boxes from your requirements.

So please don't take it as an offensive act when I now challenge you a little bit - it's definitely not my intention to criticize anything about your project, it's just curiosity regarding some design/engineering decisions (and if a different approach could also help me improving my own designs).

From your initial post I learned that your initial motivation was to print a 4x5 camera on a smaller printer. And while your initial ideas looked somehow promising, I was already assuming that you would either give up the project, buy another (bigger) printer OR you would end up designing from scratch ;-)
Your design looks like a ported version of a classic field camera, substituting wood and metal by 3D printed plastic (there's nothing wrong with such an approach). 
Which makes me a bit wondering if some of my features (integrated tripod plate as stabilizer, dovetails for stability, using simple spring steel wire) look a bit too "unconventional". 
Once again, for me it's more about learning how others approach such a design and how they handle all those different challenges (e.g. stability, printability, material substitution, tolerances).

**Reply by Egorarchitec** [reply]
First, thank you for your comment. And I certainly do not take as an offense on my design. And I agree with you. Designing the product is a lot of work, some would say what is there do design, because at the end it is just a box with a lid, but oh no... not really :D Most challenging part was to accomodate to such small scale. I am an architect/civil engineer we too work with mm in mind, but here you work with fractions of mm sometimes and what you see on a screen is much smaller, than I used to work with :D That was much fun for sure!

My initial thought to just modify your design to be able to print it on 180x180x180 3D printer transformed to slightly different idea to design a field camera that can be printed on such printer size wise and would have the maximum amount of 3D printed parts and that the non-printable parts would be as straight forward as it can be. I mean you do not need to shorten a steel lead rod, that you need tools for that not everyone has at home, the work with a stretch band is much easier than with a steel wire etc.

I see what you mean by ported version of conventional camera, and I certainly was inspired by some detail of such cameras. More like I looked how it is done on existed cameras and tried to think of the ways I can do it in my camera in PLA. So, there was a lot of thinking process for every detail of my design. I still do think about improvements and different iterations of my project and want to create a side version of my design that would be smaller in closed state but would have same front standard drop, rise, shift specs as current one (30mm drop, 40mm rise, 40 mm shift) and even would have swing back as well (maybe). My initial design was to avoid even slightest number of screws and nuts and anything you can't print. Fortunately, I backup from it on early stages as I realised how unreliable printed screws are.

As for your design features such as dovetails, stabilisation plate and so on. One side of it was a visual part, I wanted it to be a bit simpler than your design, to be more minimalistic, without much to stick out of the main body, especially at the bottom where counterpart of dovetail is in your design. The flat bottom was important for me. I didn't want to have external stabilisation as it would be an alien object on simple box, so I integrated the inner hinges with lock that work really well. For the backplate I tried different options - the full plastic clamp (like what intrepid cameras have, but of course they have it in steel), didn't work as promised, so I opted to a stretch band as it is easy to get, easy to work with, easy to cut, easy to replace.
Another side as why I am started to work on my own design from scratch is I started to think about selling it in case of success either as "hardware kit" to print and assemble with sent nonprintable parts, or just digital version of a camera or even the full assembled with ground glass and bellows. I didn't want to use your project for that obviously, so I opted to create it from the beginning, so I do not violate any license agreements or whatnot.
That would answer the question about sharing it online. As for now I do not have an intent to share it as it is still in prototype and not a final revision. But when it will be the final revision, I will try to arrange the things around to be able to sell it - find a source or a way to make my own ground glass, find a way or source to make bellows (and design the frame for them so I do not use the design from Standard Cameras). 

By the way I have a discovery about bellows from Standard Cameras that you too shell take into consideration. They changed the design of their back frame, they removed the holes on backplate to mount it through the body. They now have lip at the bottom and coule of screws that go in front of the frame (touching and pusing it into place) instead of going through the frame. I have a feeling, taht your design counted on those 4 holes (or indents) in the back frame same way as in front frame to fix it in the body. You should probably change it a bit so you can actually mount it inside. I guess they changed it when they designed their 2.0 camera.

**Comment by Nitrous2000** [comment]
This really is very impressive!  I wonder if some well placed ‘rack and pinion’ brass might add to its utility (although if your idea is 100% FDM printed 4x5, that might not appeal to you) 
are you going to make the files available?  Have you made these printer files parametric? 
  
again, great work  
     

-----Original Message-----

From: Egorarchitec ***@***.***>
To: edgarkech ***@***.***>
Cc: Subscribed ***@***.***>
Date: Monday, 9 October 2023 9:27 PM BST
Subject: Re: [edgarkech/scalable_field_camera] 4x5 modification for smaller printers (Prusa mini) (Discussion #14)

 

  Hi! Coming with an update. After initial idea to just modify the case a bit I decided to create my own design from scratch in Fusion 360. I am adding some screenshots and photos of printed prototype. I am dealing with light leaks on the back, but i think I solved it with velvet paper around the seams.





 The photo taken on this camera

**Reply by Egorarchitec** [reply]
I would love to be able to use rack and pinion made from metal instead of 3d printed. But such things are not readily available in exact dimensions in stores. The smallest stock gear wheel that I could find locally was much bigger, that i needed it to be. Rack is the same, you either should buy it bigger and cut it to your needs, or you need to order custom rack and gear you your dimensions. But ordering is either really expensive for 1 piece or you need to buy a whole batch. It is indeed interesting topic for commercial project where I can be sure that those cameras will be bought so it is logical to order a batch of those and put it in the design.

To the question of parametric files. Kinda yes. I mean Fusion 360 is parametric. But the design is quite complex (and I am not as sufficient in it as I would hope to be) so the parameters are all over the place. Changing one can lead to all kind of errors. But due to timeline it is quite easy to go through them one by one. But is definitely not scalable from 4x5 to 8x10 in one parameter change.

And another reason I started to do it from the beginning is to design it with tolerances from the beginning. Every part is divided from another by 0,2mm and it works really well. You said that you designed it without those gaps and needed post production and sanding, so I wanted to eliminate it. It basically do not need any post production other than removing supports from some parts after printing.
